src/framework/global/thirdparty/tinyxml/tinyxml2.cpp

```diff
@@ -23,9 +23,7 @@ distribution.
 
 #include "tinyxml2.h"
 
-#include "mu_patch.h"

```

Comment:
This seems to indicate that we patched it at some point. See https://github.com/musescore/MuseScore/commit/d6a586432e2cbd7b3ce03692e3307119c1e3ea5d. We should check if that patch is still needed.

Reply:
I think this is the reason for the failing tests. I added the line again.

Reply:
But note that it's not only that line, that's why I mentioned https://github.com/musescore/MuseScore/commit/d6a586432e2cbd7b3ce03692e3307119c1e3ea5d